config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+dr7ZPj4CHkEVnnuOF7mVBspE2U2nhpiEvhgQEFb3TSZe7e4LSzAMOPpmgP2kFmibCs4iBghWFe8YZgUhwMKnuS+Wvb2TNhKMxMljhYCavRHY/36I3ZWtWB8cfr0+sNNtggRo/l5UMNg/ilkcH9jNpxExUo7OZH/iBFUiPnCWRsWGCmysl18DQ1QxMbDEKfzE3pgyH/5fOQPqHtiRGev1p0Ws6wLVaW8FsVxu512SsSZGfrq3zNzM5BSu7NHPnbjrllCz9T4OmsJCCxQ75g8nB9v+4kXX1A+3QAOUeiJDm1QXVZl6NJwpbBmrkb+z1fnW24ju9IflMlZ7rHIhtYPoAM3wD4y8T0wlNRGKGuf8kC7OiCCHY378+qSdaLA4xbY6U/Bh2v9zap6bKAmytbv7M3IERcMX--ZPwPZgxh/6UY+c4y--q2OB3IoKEzGXJgD2nbRO2w==
```

Comment:
where we are using it. and why rails generating it?

Reply:
Bhai, we are using `credentials.yml.enc` as it is an encrypted file that securely stores these sensitive credentials (like API keys, database credentials, and third-party service credentials), ensuring they are protected from exposure, even if the repository becomes public or shared unintentionally. 
Rails generates this file to encourage best practices for managing sensitive data securely and for maintaining both convenience and security.